snmp/datadog_checks/snmp/data/default_profiles/eaton-epdu.yaml

```diff
@@ -325,8 +325,6 @@ metrics:
     symbols:
       - name: eaton.epdu.outletVA
         OID: 1.3.6.1.4.1.534.6.6.7.6.5.1.2
-      - name: eaton.epdu.outletCurrentPercentLoad
-        OID: 1.3.6.1.4.1.534.6.6.7.6.4.1.10
```

Comment:
why was this metric removed ?

Reply:
it's from a different table ([duplication](https://github.com/DataDog/integrations-core/blob/35c4dd137092dd5f7ee27e8a4cead969f8a8c6bb/snmp/datadog_checks/snmp/data/default_profiles/eaton-epdu.yaml#L297-L305)), we discussed it on slack a while ago